core/builtins/__init__.py

```
@@ -88,20 +94,22 @@ def __init__(self, target_from, target_id, sender_from=None, sender_id=None):
-            self.parent.target.sender_info = exports.get("BotDBUtil").SenderInfo(f'{sender_from}|{sender_id}')
+            self.parent.target.sender_id = exports.get("BotDBUtil").SenderInfo(f'{sender_from}|{sender_id}')
 
 
 Bot.FetchedSession = FetchedSession
 
 
 class FetchTarget(FetchTarget):
     @staticmethod
-    async def post_global_message(message, user_list: List[FetchedSession] = None, i18n=False, **kwargs):
+    async def post_global_message(message: str,
+                                  user_list: List[FetchedSession] = [],
```

Ruff (0.8.0)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)